docs/framework/whats-new/index.md

```diff
@@ -164,44 +164,40 @@ The .NET Framework 4.6.2 includes new features in the following areas:
   
 -   [Debugging improvements](#Debug462)  
   
- For a list of new APIs added to the .NET Framework 4.6.2, see [.NET Framework 4.6.2 API Changes](https://github.com/Microsoft/dotnet/blob/master/releases/net462/dotnet462-api-changes.md) on GitHub. For a list of feature improvements and bug fixes in the .NET Framework 4.6.2, see [.NET Framework 4.6.2 API Changes](http://go.microsoft.com/fwlink/?LinkId=708778) on GitHub.  For additional information, see [Announcing .NET Framework 4.6.2](https://blogs.msdn.microsoft.com/dotnet/2016/08/02/announcing-net-framework-4-6-2/) in the .NET Blog.  
+ For a list of new APIs added to the [!INCLUDE[net_v462](../../../includes/net-v462-md.md)], see [.NET Framework 4.6.2 API Changes](https://github.com/Microsoft/dotnet/blob/master/releases/net462/dotnet462-api-changes.md) on GitHub. For a list of feature improvements and bug fixes in the [!INCLUDE[net_v462](../../../includes/net-v462-md.md)], see [.NET Framework 4.6.2 API Changes](http://go.microsoft.com/fwlink/?LinkId=708778) on GitHub.  For additional information, see [Announcing .NET Framework 4.6.2](https://blogs.msdn.microsoft.com/dotnet/2016/08/02/announcing-net-framework-4-6-2/) in the .NET Blog.  
```

Comment:
a lot of the migrated topics suffer from extra spaces in the beginning and in the end